wemake_python_styleguide/violations/complexity.py

```diff
@@ -1402,22 +1402,25 @@ class TooManyMatchCaseViolation(ASTViolation):
 
 
 @final
-class ComplexFinallyViolation(ASTViolation):
+class TooLongFinallyBodyViolation(ASTViolation):
     """
-    Forbids complex ``finally`` block.
+    Forbid ``finally`` blocks with bodies that are too long.
 
     Reasoning:
         ``finally`` is very special. It executes code in all
         cases and therefore can't fail. When there are many lines in ``finally``
         it indicates a larger problem: brittle and complex cleanups.
 
     Solution:
-        Simplify the ``finally`` block. Use context managers, use ``ExitStack``.
+        Move things out of the ``finally`` block or create new functions.
+        The fewer lines you have in your ``finally`` block -
+        the safer you are from accidental errors.
+        Use context managers, use ``ExitStack``.
 
     Configuration:
-        This rule is configurable with ``--max-lines-in-finally``.
+        This rule is configurable with ``--max-finally-body-length``.
```

Comment:
We can solve this problem with better docs. We can explain what "line" means here. It basically means "statement"